tcms/rpc/tests/test_testrun.py

```diff
@@ -106,6 +106,56 @@ def test_should_remove_a_case_run(self):
         self.assertFalse(TestExecution.objects.filter(pk=self.test_execution.pk).exists())
 
 
+class TestGetCases(APITestCase):
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.plan = TestPlanFactory()
+
+        self.test_case = TestCaseFactory()
+        self.test_case.save()
```

Comment:
.save() is redundant.

Reply:
it is required for generating test case history.

Reply:
Do you need the history here ?

Reply:
if without .save(), this failure `Internal error: HistoricalTestCase matching query does not exist.` is raised.